### PR TITLE
Use get-port instead of getport

### PR DESCRIPTION
**Pull request by soyuka**
This fixes my issue.

If you want me to add a test lmk.

### CODE REVIEW

**Comment by yoshuawuyts** [comment]
Tests would be good! I'll take a look on Monday

**Comment by soyuka** [comment]
Will add tests monday!

**Comment by goto-bus-stop** [comment]
@soyuka sorry, what issue is solved by this? / what's the difference between those modules?

**Comment by soyuka** [comment]
@goto-bus-stop Meh, dunno. I'm running nginx on the bankai default port and it crashes on launch :|. I made an issue about this last month. Lmk if you want me to rebase!

**Comment by goto-bus-stop** [comment]
Ohh I see, it's #292. I think this should be fine, will check in the afternoon

**Comment by goto-bus-stop** [comment]
oh, so get-port tries to create a server while getport tries to connect to a port. i don't know when that would yield different results but if it does it makes more sense for bankai to use the former. thanks for the rebase, sorry this had to wait so long!

it looks like get-port returns a Promise and doesn't accept a callback though, could you update that? and also get rid of the package-lock please :) thanks 🙏

**Comment by soyuka** [comment]
Done, don't worry about the waiting time haha

File: lib/cmd-start.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ function start (entry, opts) {
     })
   })
 
-  getPort(8080, 9000, function (err, port) {
+  getPort({port: 8080}).then(function (err, port) {
```
Comment (goto-bus-stop) [inline lib/cmd-start.js]: just one last thing: these should be `.then(function (port) { }).catch(function (err) {})`

Comment (soyuka) [reply lib/cmd-start.js]: I should not try to fix things while in holidays \o/

**Comment by goto-bus-stop** [comment]
yay thanks!! 🎉

**Pull request merged**